packages/core/frontend/sirius-components-core/src/onboarding/RepresentationsArea.tsx

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { Collections } from '@material-ui/icons';
+
+import { RepresentationAreaProps } from './RepresentationsArea.types';
+
+const useRepresentationAreaStyles = makeStyles(() => ({
+  card: {
+    backgroundColor: 'transparent',
+
+    maxWidth: '330px',
+    minWidth: '330px',
+  },
+  cardContent: {
+    height: '100%',
+    maxHeight: '400px',
+    overflowY: 'auto',
+  },
+}));
+
+export const RepresentationsArea = ({ representations, setSelection }: RepresentationAreaProps) => {
+  const classes = useRepresentationAreaStyles();
+  // Representations list
+  let representationsButtons =
```

Comment:
- Use `const` instead of `let`
- Try to inline such simple logic in the JSX, it makes it easier to read the structure of the component